cmd/buildah/rmi.go

```diff
@@ -272,3 +284,35 @@ func storageImageID(store storage.Store, id string) (types.ImageReference, error
 	}
 	return nil, errors.Wrapf(err, "error parsing %q as a storage image reference: %v", id)
 }
+
+// Returns a list of all existing images to be removed
+func removeAllImages(store storage.Store) ([]string, error) {
```

Comment:
You're not really removing anything in this function, could we rename it to findAllImages?